tests/test_blob.py

```diff
@@ -75,6 +75,40 @@ def test_single_point():
     assert error < 1e-10, "Numerical error too big"
```

Comment:
Sure, the reason I didn't add was because I didn't find any test for the actual alignment implementation either (the current blob tests work both for alignment true and false, without changing expected values)